tests/pdctl/scheduler/scheduler_test.go

```diff
@@ -236,21 +237,21 @@ func (s *schedulerTestSuite) TestScheduler(c *C) {
 	c.Assert(roles, DeepEquals, []string{"learner"})
 
 	// test echo
```

Comment:
What about using a more clear comment here?

Reply:
added, I don't know exactly what it is testing, though.